src/lib/containers/FullWidthAlert.tsx

```diff
@@ -59,7 +59,7 @@ function ButtonFromConfig(props: {
   variant: string
 }) {
   const { config, variant, className } = props
-  if (config) {
+  if (config && ('onClick' in config || 'href' in config)) {
```

Comment:
A button with no text will be created if the config is defined but has neither `onClick` nor `href`.

Due to a mistake I made, SWC does this a lot on staging (the config object is always created), so we will probably want to patch staging's version of SRC with a build including this change.